docs/build/circuit-visualization.ipynb

```diff
@@ -4,12 +4,507 @@
+      "text/plain": [
+       "<Figure size 1058.43x869.556 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Draw the circuit without barriers and reverse bit order\n",
+    "circuit.draw(output='mpl', plot_barriers=False, reverse_bits=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Renderer-specific customizations\n",
+    "\n",
+    "Some available customizing options are specific to a renderer. The `line_length` kwarg for the `text` backend can be used to set a maximum width for the output. When a diagram is wider than the maximum, it will wrap the diagram below. The `mpl` backend has the `style` kwarg, which is used to customize the output. The `scale` option is used by the `mpl` and `latex` renderers to scale the size of the output image with a multiplicative adjustment factor. The `style` kwarg takes in a `dict` with multiple options, including specifying colors, changing rendered text for different types of gates, choosing different line styles, and more. Available options are:\n",
```

Comment:
This all being in a single paragraph is tough to read. Maybe something like a table or per-backend list would work?

Alternatively, this seems more like API docs material!

Reply:
For now I'm separating each line to see how digestible that is